AndelaOSP/andela-societies-frontend
# Ft 158792482 Display Categories View for Success Ops

### Description
This PR introduces functionality to allow success ops to view all categories, delete and add categories to be used in logging activities

### How to test
Once logged in as either a success ops team member
Navigate to Categories tab on the Sidebar

* You should view a list of categories with their respective names, description and points.
<img width="1440" alt="screen shot 2018-07-13 at 23 35 05" src="https://user-images.githubusercontent.com/6057761/42713321-b901be14-86f7-11e8-9da3-53626bb3e498.png">
* You should be able to create a category
<img width="1440" alt="screen shot 2018-07-13 at 23 35 24" src="https://user-images.githubusercontent.com/6057761/42713354-dcf7ee38-86f7-11e8-9838-4b0e9f6c1600.png">
